tests/openedx_learning/apps/authoring/contents/test_file_storage.py

```diff
@@ -0,0 +1,79 @@
+"""
+Tests for file-backed Content
+"""
+from datetime import datetime, timezone
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.test import override_settings
+
+from openedx_learning.apps.authoring.contents import api as contents_api
+from openedx_learning.apps.authoring.contents.models import get_storage
+from openedx_learning.apps.authoring.publishing import api as publishing_api
+from openedx_learning.lib.test_utils import TestCase
+
+
+class ContentFileStorageTestCase(TestCase):
+    """
+    Test the storage of files backing Content.
+    """
+    def setUp(self) -> None:
+        """
+        It's actually important that we use setUp and not setUpTestData here,
+        because at least one of our tests will clear the get_storage cache,
+        meaning that subsequent tests will get a new instance of the
+        InMemoryStorage backend–and consequently wouldn't see any data loaded
+        by setUpTestData.
+
+        Recreating the test data for every test lets individual tests change the
+        storage configuration without creating side-effects for other tests.
+        """
+        super().setUp()
+        learning_package = publishing_api.create_learning_package(
+            key="ContentFileStorageTestCase-test-key",
+            title="Content File Storage Test Case Learning Package",
+        )
+        self.html_media_type = contents_api.get_or_create_media_type("text/html")
+        self.html_content = contents_api.get_or_create_file_content(
+            learning_package.id,
+            self.html_media_type.id,
+            data=b"<html>hello world!</html>",
+            created=datetime.now(tz=timezone.utc),
+        )
+
+    def test_file_path(self):
+        """
+        Test that the file path doesn't change.
+
+        If this test breaks, it means that we've changed the convention for
+        where we're storing the backing files for Content, which means we'll be
+        breaking backwards compatibility for everyone. Please be very careful if
+        you're updating this test.
+        """
+        content = self.html_content
+        assert content.file_path() == f"content/{content.learning_package.uuid}/{content.hash_digest}"
```

Comment:
I feel like it would be better to hard-code a UUID and hash digest into the string here, because I could certainly imagine the UUID (and to a lesser extent the hash digest) could be serialized in different ways, breaking backwards compatibility, but not breaking the test.

`545fc5c8-5f07-442f-9f69-69b936d82d51` vs `545fc5c85f07442f9f6969b936d82d51` vs `545FC5C8-5F07-442F-9F69-69B936D82D51` are all the same UUID.

Though I'm not sure if you have a way to create entities with a specific UUID during tests so maybe we can't.

Reply:
We don't right now... we probably should, if we want to accurately re-create from an import. But I don't want to add that to that to the scope of this PR.